gapis/api/vulkan/vulkan.api

```diff
@@ -2623,6 +2634,8 @@ cmd VkResult vkGetPhysicalDeviceImageFormatProperties(
     VkImageUsageFlags        usage,
     VkImageCreateFlags       flags,
     VkImageFormatProperties* pImageFormatProperties) {
+  assert(notNullOrZero(physicalDevice))
```

Comment:
These aren't a good cases for `assert`. `assert` should be used for internal sanity checking. You're trying to use `assert` where proper error checking should be used. Check the parameters and raise a human readable error message.

In GLES we have [errors.go](https://github.com/google/gapid/blob/master/gapis/api/gles/api/errors.api) for raising and reporting human readable errors. This still needs a lot of work:

a) The error messages should include the parameter name. Tracked by #1149.
b) We should make the externs being used part of the API language.
c) The way the stringtable entries are linked in is horrible. Again, we should make these 1st class language features.

That said, if you keep the implementation as close to GLES as possible, I can clean up both together.